### PR TITLE
[ET-1883] ET > TC > Attendee List Export for Tickets Commerce does not populate order status

### PR DESCRIPTION
**Pull request by redscar**
### 🎫 Ticket

[ET-1883] 

### 🗒️ Description

The `order_status` column was being referenced incorrectly as `order_status_label`.

### 🎥 Artifacts 

[Event-with-ARF-attendees(2).csv](https://github.com/the-events-calendar/event-tickets/files/13834762/Event-with-ARF-attendees.2.csv)

### ✔️ Checklist
- [x] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

[ET-1883]: https://stellarwp.atlassian.net/browse/ET-1883?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by estebanhdzc — APPROVED** [review]
Nicely done!

File: src/Tribe/Attendees.php [hunk 1]
```diff
@@ -551,13 +551,47 @@ private function generate_filtered_list( $event_id ) {
...
+			'event-tickets'
+		);
+		$export_columns['order_status'] = esc_html_x(
+			'Order Status',
+			'attendee export',
+			'event-tickets'
+		);
+		$export_columns['attendee_id']  = esc_html(
+			sprintf(
+			/* translators: %s: The type of ID. */
```
Comment (codingmusician) [inline src/Tribe/Attendees.php:566]:
```suggestion
				/* Translators: %s: The type of ID. */
```

**Review by rafsuntaskin — CHANGES_REQUESTED** [review]
@redscar Can you please confirm that these changes are not impacting the exported data from Woo or EDD. I assume `order_status_label` is still used by them, so completely removing that might have an impact.

**Comment by redscar** [comment]
> @redscar Can you please confirm that these changes are not impacting the exported data from Woo or EDD. I assume `order_status_label` is still used by them, so completely removing that might have an impact.

Thanks @rafsuntaskin. Good call, I just tested it locally and everything looks to be working properly, even with this change.

**Review by rafsuntaskin — APPROVED** [review]

**Pull request merged**